hibernate-cache/src/test/java/org/infinispan/test/hibernate/cache/functional/cluster/NaturalIdInvalidationTest.java

```diff
@@ -41,6 +42,9 @@
 
 	private static final InfinispanMessageLogger log = InfinispanMessageLogger.Provider.getLog(NaturalIdInvalidationTest.class);
 
+	@Rule
```

Comment:
Indentation is fine. 3 spaces.

The rest of indentation is tabs, so you as in the previous comment, adjust the tab to be 3 spaces passing in `?ts=3`